src/NuGet.Core/NuGet.ProjectModel/FileFormatException.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace NuGet.ProjectModel
 {
+    [SerializableAttribute]
     public class FileFormatException : Exception
```

Comment:
This class needs to implement `ISerializable` properly for the properties to round-trip.